tests/test-map.c

```diff
@@ -9,7 +9,23 @@ static int compare_fn_int(void *x, void *y) {
 	return 0;
 }
 
-void test_map(void) {
+bool predicate_fn_one(void *key) {
+	if (*(int *)key == 20) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+bool predicate_fn_two(void *key) {
+	if (*(int *)key == 30) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+void test_main(void) {
```

Comment:
change this back to `test_map`

**Remember**: You should only add the tests inside this block and *do not* touch the function name/function signature/CMake files and such.